looker/definitions/ads.toml

```diff
@@ -34,6 +34,48 @@ description = "Privacy-Preserving Attribution (PPA) conversions"
 [metrics.spoc_clicks.statistics.sum]
 [metrics.spoc_revenue.statistics.sum]
 
+[metrics.newtab_spend]
+select_expression = "SUM(spend)"
+data_source = "newtab_desktop_ad_metrics"
+friendly_name = "Spend"
+description ="Newtab Desktop Metrics spend"
+
+[metrics.newtab_dismisses]
+select_expression = "SUM(dismisses)"
+data_source = "newtab_desktop_ad_metrics"
+friendly_name = "Dismisses"
```

Comment:
specify that these are desktop dismisses in the friendly name

Reply:
Will do! Thanks for the feedback @jaredsnyder